package com.ky.exceptions;

public class ThrowsDemo2 {
    public static void main(String[] args) throws ArithmeticException{
        ThrowsDemo th=new ThrowsDemo();
        th.calc(10);
    }
}
